department-of-veterans-affairs/va.gov-cms
# VACMS-5888: Hide existing VC services from option list to prevent duplicates.

## Description

Closes #5888

## Testing done
Visual

## QA steps
- As an administrator, go to `node/3756/edit`.
- Add a service, and visually verify only options that are not selected in the table above appear.
- Remove items from the table, and verify that they appear in the select.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
